ansible/image-upload.yml

```diff
@@ -0,0 +1,45 @@
+---
+
+- name: Push Collector Image to Hosts
+  vars_files:
+    - group_vars/all.yml
+  hosts: "platform_*:&job_id_{{ job_id }}"
+  strategy: free
+
+  tasks:
+    - name: Get Collector Image
+      shell: make -C "{{ collector_root }}" tag
+      register: collector_tag_result
+      delegate_to: localhost
```

Comment:
Could we instead control the image tag being used with an env variable? Sometimes you want to use an existing image you built a couple days back, but might not have the branch in the same state anymore.

Reply:
yeah that's a good idea. I'll make it default to the current tag if the env var is not provided